content/general/channel-rules-namespaces.textile

```diff
@@ -1,13 +1,7 @@
 ---
 title: Channel Rules and Namespaces
+section: general
 index: 15
 ---
 
-Apps can have one or more channel rules with configurable settings that will be applied to matching channels created in the app.  If no channel rule is found when a channel is created, the default channel rule for the app is applied.
-
-Channel rules allow settings such as whether messages are persisted or TLS required to be configured.  The name in each configured channel rule will match any channel with that name or any channel in that namespace.  For example, a channel rule with the configured namespace of "rss" will match the channel name "rss" as well as the channel name "rss:news" which is a channel within the "rss" namespace.
-
-Each channel rule has the following configurable settings:
-* **Persisted messages** - If enabled, all messages within this namespace will be persisted. You can access stored messages via the History API. The number of hours a message is stored is configurable, "find out more about message persistence":http://support.ably.com/solution/articles/3000030059-how-long-are-messages-stored-for
```

Comment:
Could the following Hubspot equivalent of the support article be used instead: (https://knowledge.ably.com/how-long-are-messages-stored-for-1)  ? In general we want to phase out the use of Freshdesk support article references as we are moving away from using it

Reply:
This is in red, meaning it's being deleted.

More generally -- there are 191 links to support articles in the docs repo. They (presumably!) were imported with predictable urls. Surely when we decide to switch over, we will replace the links all at once with a batch find/replace job. I don't think a switchover plan of 'manually replace the links one by one as they turn up in PRs over the next 10 years, and in the meantime have a mixture of the two' is practical.

Reply:
Thanks Simon and true that will not be ideal and all of the titles of the Hubspot articles are the same as those in Freshdesk so there shouldn't be a need to resort to manually moving everything over

Reply:
I am also thinking that we should update this support article - @markcoxsmith can you add section to it that asks about long term persistence of last message? 

Feel free to loop me in if you need help with the copy.